internal/devserver/main/main.go

```diff
@@ -32,7 +41,65 @@ func main() {
 		os.Exit(1)
 	}
 
-	http.Handle(*servingPath, concatjs.ServeConcatenatedJS(*manifest, *base, nil /* realFileSystem */))
+	scriptsManifestPath := filepath.Join(*base, *scriptsManifest)
+	scriptFiles, err := manifestFiles(scriptsManifestPath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to read scripts_manifest: %v\n", err)
+		os.Exit(1)
+	}
+
+	preScripts := make([]string, 0, 100)
+	postScripts := make([]string, 0, 1)
+
+	// Include the livereload script if IBAZEL_LIVERELOAD_URL is set and
+	// valid and the script is available. Also output a snippet before the
+	// livereload script that sets window.LiveReloadOptions which configures
+	// the endpoint livereload will use
+	livereloadUrl := os.Getenv("IBAZEL_LIVERELOAD_URL")
+	re := regexp.MustCompile("^([a-zA-Z0-9]+)\\:\\/\\/([[a-zA-Z0-9\\.]+)\\:([0-9]+)")
+	match := re.FindStringSubmatch(livereloadUrl)
+	if match != nil && len(match) == 4 {
+		port, err := strconv.ParseUint(match[3], 10, 16)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Cannot determine livereload port from IBAZEL_LIVERELOAD_URL")
+		} else {
+			livereloadScheme := match[1]
+			livereloadHost := match[2]
+			livereloadPort := uint16(port)
+			if *livereloadScript != "" {
+				livereloadJs, err := loadScript(filepath.Join(*base, *livereloadScript))
```

Comment:
see above, I think it's easier to have `ibazel` deal with all aspects of the live reloading, including serving the script from disk (or whereever actually).